css/styles.css

```diff
@@ -0,0 +1,56 @@
+.artdeco1 {
+  margin-left: -8px;
+  margin-top: -8px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+h1 {
+  color: #1a2236;
+  width: 220px;
+  line-height: 100%;
+  margin-top: 19px;
+  margin-left: 16px;
+  font-weight: 400;
+  font-family: 'Playfair Display', serif;
+  font-size: 50px;
+}
+
+p {
+  color: #1a2236;
+  width: 288px;
+  height: 80px;
+  margin-top: 21px;
+  margin-left: 16px;
+  font-family: 'Poppins', serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+}
+
+button {
+  border-color: #1a2236;
+  font-family: 'Poppins', serif;
+  font-size: 16px;
```

Comment:
[OPTIONAL]

Consider making the font size a little bigger, maybe something in between 20px or 25px :wink: